src/main/java/teammates/ui/servlets/AuthServlet.java

```diff
@@ -50,6 +50,7 @@ AuthorizationCodeFlow getAuthorizationFlow() throws IOException {
     String getRedirectUri(HttpServletRequest req) {
         GenericUrl url = new GenericUrl(req.getRequestURL().toString().replaceFirst("^http://", "https://"));
         url.setRawPath("/oauth2callback");
+        url.set("ngsw-bypass", "true");
```

Comment:
LGTM (apart for this line here, which looks like it's from another issue?)

Reply:
Should be because I amended a commit on master. Changes have gone away after updating the branch in this PR